src/pages/workspaces/workspace/analysis/Analyses.js

```diff
@@ -247,6 +247,8 @@ const Analyses = _.flow(
   storageDetails: { googleBucketLocation, azureContainerRegion },
   onRequesterPaysError
 }, _ref) => {
+  const { ExportAnalysisModal } = analysisComponentDependencies.get()
```

Comment:
minimal edge adaption for rest of codebase that is not ready to fully participate in Composable DI framework.